test/HistoryGTest.cpp

```diff
@@ -0,0 +1,77 @@
+////////////////////////////////////////////////////////////////////////////////
+/**
+* @file HistoryGTest.cpp
+* @brief unit tests for history functions
+*/
+////////////////////////////////////////////////////////////////////////////////
+
+#include <cmath>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <string>
+#include <vector>
+
+#include "flatbuffers/idl.h"
+#include "History.h"
+
+using namespace calculator;
+using namespace testing;
+
+class whenTestingHistory:public ::testing::Test
```

Comment:
add spacing around the colon `class whenTestingHistory : public ::testing::Test`